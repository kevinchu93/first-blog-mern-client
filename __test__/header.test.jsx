import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from '../src/app/header';

test('Header component should render as expected', () => {
  expect(toJson(shallow(<Header />))).toMatchSnapshot();
});
