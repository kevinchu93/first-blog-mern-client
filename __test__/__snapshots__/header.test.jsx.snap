// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render as expected 1`] = `
<div>
  <div
    className="header"
  >
    <h1
      className="header__title"
    >
      My First Blog
    </h1>
  </div>
  <div
    className="header__nav"
  >
    <Link
      className="header__nav__link"
      replace={false}
      to="/posts"
    >
      POSTS
    </Link>
    <Link
      className="header__nav__link"
      replace={false}
      to="/post_new"
    >
      NEW
    </Link>
    <Link
      className="header__nav__link"
      replace={false}
      to="/info"
    >
      INFO
    </Link>
  </div>
</div>
`;
